fmriprep/workflows/anatomical.py

```diff
@@ -74,43 +74,44 @@ def t1w_preprocessing(settings, name='t1w_preprocessing'):
             testing=settings.get('debug', False),
             template='mni_icbm152_nlin_asym_09c'
         ),
-        name='T1_2_MNI_Registration'
+        name='t1_2_mni'
     )
     # should not be necesssary but does not hurt - make sure the multiproc
     # scheduler knows the resource limits
     t1_2_mni.interface.num_threads = settings['ants_nthreads']
 
     # 6. FreeSurfer reconstruction
     if settings['freesurfer']:
-        surface_recon = surface_reconstruction(settings=settings)
+        surface_recon = surface_reconstruction(name='surface_recon',
```

Comment:
What about:

* The variable name should be a substring or superstring of the function name
* The workflow name should be the function name if only one is made, the variable name if multiple

Then we have either:

```Python
surface_recon = surface_reconstruction(name='surface_reconstruction')
```
or
```Python
surface_recon1 = surface_reconstruction(name='surface_recon1')
surface_recon2 = surface_reconstruction(name='surface_recon2')
```